### ISSUE TITLE
Focus on text field but enters hotkey

### ISSUE THREAD

**Issue by gregfenton** [JohannesKlauss/react-hotkeys-hook#802]
The following example:

**REPRO CODE:**
```js
function ExampleComponent() {
  useHotkeys('/', () => inputRef.current.focus());
  const inputRef = useRef();

  return (
    <input type="text" ref={inputRef} placeholder="search via '/'" />
  )
}
```

**EXPECTED BEHAVIOUR:**
- without a focus in the <input> field, typing `/` would focus in the field so that a search string could be entered

**ACTUAL BEHAVIOUR:**
- without focus in the <input> field, typing '/' puts focus in the field and sets its initial value to `/`

I could not determine from the documentation if there is an option to suppress propagating the initial entry of the hotkey?

**Related ask:**  Is there a documented example for having a hotkey such as "/" be used to focus to an input field, but also allow "/" to be entered once the field has focus?  That is, if the field does not have input and I type "/abc" then the search field gets focus and the text value becomes "abc".  But if the field has focus and I type "abc/def" then the field gets the value "abc/def".

**Comment by gregfenton** [comment]
I got a working example of what I think is valid behaviour.  But it is more code that I'd expect to have to do.  Wondering if either:
- I'm missing some capability of react-hotkeys-hook I should be aware of?
- This example is a potential "new feature" for react-hotkeys-hook ?

```js
function ExampleComponent() {
  const HOTKEY = '/';
  const [enableHotkey, setEnableHotkey] = useState(true);

  const handleHotkeyInput = () => {
    inputRef.current.focus();
    setEnableHotkey(false);
  }

  useHotkeys(HOTKEY, handleHotkeyInput, {enableHotkey});
  const inputRef = useRef();

  const handleInputChange = () => {
    if (!enableHotkey) {
      if (inputRef.current.value === HOTKEY) {
        inputRef.current.value = "";
      }
    }
  };

  return (
    <>
    <input type="text" ref={inputRef} onChange={handleInputChange} onBlur={() => setEnableHotkey(true)} />
    <button onClick={() => console.log("inputRef value", inputRef.current.value, enableHotkey)}>foo</button>
    </>
  )
}
```

**Comment by JohannesKlauss** [comment]
I'll look into it, thanks for raising the issue

**Comment by JohannesKlauss** [comment]
Fixed in 4.3.1

Example Code:

```tsx
function App() {
  useHotkeys('shift+/', (e) => inputRef.current?.focus(), { preventDefault: true })

  const inputRef = useRef<HTMLInputElement>(null)

  return (
    <input type='text' ref={inputRef} placeholder="search via '/'" />
  )
}
```

Mind you that in newer versions you have to use `shift+/` instead of `/`.
